.github/workflows/publish.yml

```diff
@@ -1,21 +1,52 @@
-name: publish
+name: Build and Publish Docker Image
 
 on:
   push:
-    branches: [master, develop]
+    branches:
+        - "main"
```

Comment:
Should this be "master"? We've talked about switching "master" to "main" but we haven't implemented that yet.

Also, I think there was a suggestion to only build a new Docker image on "master" when a new version tag was added. Is there a downside to that approach?

Reply:
Yep! Good catch. Fixed.

Reply:
> only build a new Docker image on "master" when a new version tag was added. Is there a downside to that approach

I did not do that here. They are treated as separate events. "They" being:
- merging into `master`
- adding a semver git tag